testdata/cluster/internal_outbound.yaml.tmpl

```diff
@@ -11,8 +11,8 @@ load_assignment:
       metadata:
         filter_metadata:
           tunnel:
-            address: 127.0.0.1:{{ .Ports.ServerTunnelPort }}
-            destination: 127.0.0.1:{{ .Ports.ServerPort }}
+            address: 127.0.0.2:{{ .Ports.ServerTunnelPort }}
```

Comment:
Any background/context information for using 127.0.0.2?

Reply:
Ambient uses IP-based lookup, will simplify testing when client and server have different IPs